iOS/DuckDuckGo/AutofillLoginListViewModel.swift

```diff
@@ -458,10 +458,10 @@ final class AutofillLoginListViewModel: ObservableObject {
     private func updateViewState() {
         var newViewState: AutofillLoginListViewModel.ViewState
         
-        if authenticator.state == .loggedOut && !authenticationNotRequired {
-            newViewState = .authLocked
-        } else if authenticator.state == .notAvailable {
+        if !authenticator.canAuthenticate() {
```

Comment:
While testing I also discovered that the incorrect UI state was being presented on the Passwords screen for users who do not have any device security enabled, which I’ve fixed here